api/v1/sriovnetworknodestate_types.go

```diff
@@ -26,6 +26,7 @@ import (
 // SriovNetworkNodeStateSpec defines the desired state of SriovNetworkNodeState
 type SriovNetworkNodeStateSpec struct {
 	Interfaces Interfaces `json:"interfaces,omitempty"`
+	RdmaMode   string     `json:"rdmaMode,omitempty"`
```

Comment:
we need to talk about this one in the community I think we should have an higher level object first something like System/Host and we have that inside

Reply:
done